static/app/views/performance/transactionSummary/transactionEvents/content.spec.tsx

```diff
@@ -188,17 +173,20 @@ describe('Performance Transaction Events Content', function () {
           />
         </OrganizationContext.Provider>
       </RouteContext.Provider>,
-      initialData.routerContext
+      {context: initialData.routerContext}
     );
-    await tick();
-    wrapper.update();
 
-    expect(wrapper.find('EventsTable')).toHaveLength(1);
-    expect(wrapper.find('CompactSelect')).toHaveLength(1);
-    expect(wrapper.find('StyledSearchBar')).toHaveLength(1);
-    expect(wrapper.find('Filter')).toHaveLength(1);
+    expect(screen.getByTestId('events-table')).toBeInTheDocument();
+    expect(screen.getByText(textWithMarkupMatcher('Percentilep100'))).toBeInTheDocument();
+    expect(
+      screen.getByPlaceholderText(t('Search for events, users, tags, and more'))
+    ).toBeInTheDocument();
+    expect(screen.getByTestId('span-operation-breakdown-filter')).toBeInTheDocument();
 
-    const columnTitles = wrapper.find('EventsTable').props().columnTitles;
+    const eventsTable = screen.getByTestId('events-table');
+    const columnTitles = Array.from(eventsTable.querySelectorAll('th')).map(
+      elem => elem.textContent
+    );
```

Comment:
```suggestion
    const columnTitles = screen
      .getAllByRole('columnheader')
      .map(elem => elem.textContent);
```

Reply:
omg, yes this is what i needed. thank you @priscilawebdev!